### DISCUSSION TITLE
Spacy - Security scanning

### DISCUSSION THREAD

**Discussion by rangawillb4u** [explosion/spaCy#10630] [Help: Best practices]
Just wanted to check. Is the spacy package scanned for any vulnerabilities prior to releasing new versions?

**Comment by adrianeboyd** [comment]
No, it's not currently. We should set up `SECURITY.md` and see if there are scanning options that would make sense for our repos.

**Comment by rangawillb4u** [comment]
Thank you Adriane  for the response 👍
